docs/tools-reference/custom-kernels-distros/install-a-custom-distribution-on-a-linode/index.md

```diff
@@ -45,16 +45,28 @@ In this section you'll install your custom distro onto a raw disk, with the *dir
 
 ### Prepare your Linode
 
+1.  Log into the [Cloud Manager](https://cloud.linode.com) and create a Linode. For this example, since you are going to install Debian 8, we chose Debian 9 so it's easy to see that the installation is successful. You can use a Nanode for this guide.
```

Comment:
This is unclear: "For this example, since you are going to install Debian 8, we chose Debian 9." This might have to do with the instructions toward the beginning of the guide that say you are going to use Debian 8, but then you use Debian 9. So is the idea to create a Linode with Debian 8, then install a custom distro of Debian 9 on top of that?